.github/workflows/docker-build-push.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish Docker image
```

Comment:
I would make it `Publish Docker Image` or `Publish docker image`

Reply:
I have implemented this change.